man/journald.conf.xml

```diff
@@ -120,18 +120,26 @@
       <varlistentry>
         <term><varname>SplitMode=</varname></term>
 
-        <listitem><para>Controls whether to split up journal files per user, either <literal>uid</literal> or
-        <literal>none</literal>. Split journal files are primarily useful for access control: on UNIX/Linux access
-        control is managed per file, and the journal daemon will assign users read access to their journal files. If
-        <literal>uid</literal>, all regular users (with UID outside the range of system users, dynamic service users,
-        and the nobody user) will each get their own journal files, and system users will log to the system journal.
-        See <ulink url="https://systemd.io/UIDS-GIDS">Users, Groups, UIDs and GIDs on systemd systems</ulink>
-        for more details about UID ranges.
-        If <literal>none</literal>, journal files are not split up by user and all messages are
-        instead stored in the single system journal. In this mode unprivileged users generally do not have access to
-        their own log data. Note that splitting up journal files by user is only available for journals stored
-        persistently. If journals are stored on volatile storage (see <varname>Storage=</varname> above), only a single
-        journal file is used. Defaults to <literal>uid</literal>.</para></listitem>
+        <listitem><para>Controls whether to split up persistent journal files per user, either
+        <literal>uid</literal> or <literal>none</literal>. Split journal files are primarily useful for
+        access control: on UNIX/Linux access control is managed per file, and the journal daemon will assign
+        users read access to their journal files. If <literal>uid</literal>, all regular users (with UID
+        outside the range of system users, dynamic service users, and the nobody user) will each get their
+        own journal files, and system users will log to the system journal.  See <ulink
+        url="https://systemd.io/UIDS-GIDS">Users, Groups, UIDs and GIDs on systemd systems</ulink> for more
+        details about UID ranges.  If <literal>none</literal>, journal files are not split up by user and all
+        messages are instead stored in the single system journal. In this mode unprivileged users generally
+        do not have access to their own log data. Defaults to <literal>uid</literal>.</para></listitem>
+      </varlistentry>
+
+      <varlistentry>
+        <term><varname>VolatileSplitMode=</varname></term>
+
+        <listitem><para>Same as <varname>SplitMode=</varname> described previously but controls whether to
+        split up volatile journal files per user. It takes the same values, i.e. either
+        <literal>uid></literal> or <literal>none</literal> and allows unprivileged users to have access to
+        their own volatile log data when set to <literal>uid</literal>. Defaults to
+        <literal>none</literal></para></listitem>
```

Comment:
Not convinced this should be a whole new setting. I don#t see any value in allowing the user to configure per-uid split mode in volatile mode and dsiable it in persistent mode.

Let's simplify this and stay with a single setting, i.e. `SplitMode=none|uid|uid-everywhere`, i.e. where `uid` means "split in persistent mode, not split in volatile mode". And `uid-everywhere` means split in both volatile and persistent mode.

Let's not go overboard with adding options over options.